templates/content-fluid-en.md

```diff
@@ -7,6 +7,8 @@ altLangPage: content-fluid-fr.html
 secondlevel: false
 dateModified: 2021-02-05
 share: true
+feedbackPath: "/sites/feedback/ajax/report-problem-en.html"
```

Comment:
Use a relative path instead. It won't work if the site run in a sub directory like GCWeb on github.
```suggestion
feedbackPath: "../sites/feedback/ajax/report-problem-en.html"
```

Reply:
Changed.Thanks